src/libUtils/DSComposition.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2019 Zilliqa
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#ifndef __DSCOMPOSITION_H__
+#define __DSCOMPOSITION_H__
+
+#include "Logger.h"
```

Comment:
Minor comments (no need to change for now)
1. I prefer just including the minimum required headers in a header file.  Logger, for example, is not needed here
2. We can probably keep this function inside libDirectoryService instead of libUtils?

Reply:
1. I've removed the unnecessary '#include "Logger.h"', thanks.
2. I can move it to libDirectoryService. The original rationale was that it was split out because libNode had an exact same copy of the code in it's own DSBlockProcessing.cpp in the current codebase and I did not want to introduce even more coupling. I moved the file to libDirectoryService.